scss/details.scss

```diff
@@ -19,3 +19,13 @@
 		}
 	}
 }
+
+.book-props {
+	display: flex;
```

Comment:
What you can do is create a `display.scss` file under the path `src/scss/common/`. In that file you can create common classes used to display different things that are very boilerplate:
```scss
.row {
  display: flex;
  flex-direction: row;
}

.column {
  display: flex;
  flex-direction: column;
}